private_gpt/settings/settings.py

```diff
@@ -217,6 +217,31 @@ class QdrantSettings(BaseModel):
     )
 
 
+class SQLDatabaseSettings(BaseModel):
+    enabled: bool = Field(
+        False,
+        description="Flag to enable SQL Query mode. Disabled by default",
+    )
+    dialect: Literal["mysql"]
+    driver: Literal["pymysql"]
+    db_host: str | None = Field(
+        "localhost",
+        description="Host name of Database server. If host is None, set to 'localhost'.",
+    )
+    db_user: str | None = Field(
+        "root",
+        description="Username to be used for accessing the SQL Database Server. If user is None, set to 'root'.",
+    )
+    db_password: str = Field(
+        "",
+        description="Password to be used for accessing the SQL Database Server. If password is None, set to empty string.",
+    )
```

Comment:
The description is misleading. The default is a blank string, which is different from `None`